extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -198,12 +220,12 @@ export class AmpSlideScroll extends BaseSlides {
 
   /** @override */
   moveSlide(dir, animate) {
-    if (this.slideIndex_ != null) {
+    if (this.slideIndex_ !== null) {
       const hasNext = this.hasNext();
       const hasPrev = this.hasPrev();
       if ((dir == 1 && hasNext) ||
           (dir == -1 && hasPrev)) {
-        let newIndex = this.slideIndex_ + dir;
+        let newIndex = (this.slideIndex_ || 0) + dir;
```

Comment:
i would rather have this error out. you could use - dev().assertNumber here as well.